R/helpers-pattern.R

```diff
@@ -174,14 +174,29 @@ matches <- function(match,
 #' @param range A sequence of integers, like `1:5`.
 #' @param width Optionally, the "width" of the numeric range. For example,
 #'   a range of 2 gives "01", a range of three "001", etc.
+#' @inheritParams rlang::args_dots_empty
+#' @param cross Whether to take the cartesian product of `prefix`, `range`, and `suffix`.
+#'   If `FALSE`, the default, these arguments are recycled using tidyverse rules.
```

Comment:
```suggestion
#'   If `FALSE`, the default, these arguments are recycled using [tidyverse rules][vctrs::vector_recycling_rules].
```

+ redocument

I think that's how the crosslink is specified

Historically people in dplyr have asked me to cross link to the recycling rules because they are like "well, what are they?"